modules/statics/src/coins.ts

```diff
@@ -25,10 +25,16 @@ export const coins = CoinMap.fromCoins([
   utxo('tzec', 'Testnet ZCash', Networks.test.zCash, UnderlyingAsset.ZEC),
   account('algo', 'Algorand', Networks.main.algorand, 6, UnderlyingAsset.ALGO),
   account('talgo', 'Testnet Algorand', Networks.test.algorand, 6, UnderlyingAsset.ALGO),
+  account('cgld', 'Celo Gold', Networks.main.celo, 18, UnderlyingAsset.CGLD, ETH_FEATURES),
+  account('tcgld', 'Testnet Celo Gold', Networks.test.celo, 18, UnderlyingAsset.CGLD, ETH_FEATURES),
   account('eth', 'Ethereum', Networks.main.ethereum, 18, UnderlyingAsset.ETH, ETH_FEATURES),
   account('teth', 'Testnet Ethereum', Networks.test.kovan, 18, UnderlyingAsset.ETH, ETH_FEATURES),
+  account('etc', 'Ethereum Classic', Networks.main.ethereum, 18, UnderlyingAsset.ETC, ETH_FEATURES),
+  account('tetc', 'Testnet Ethereum Classic', Networks.test.kovan, 18, UnderlyingAsset.ETC, ETH_FEATURES),
   account('eos', 'Eos', Networks.main.eos, 4, UnderlyingAsset.EOS),
   account('teos', 'Testnet Eos', Networks.test.eos, 4, UnderlyingAsset.EOS),
+  account('rbtc', 'Rsk', Networks.main.rbtc, 18, UnderlyingAsset.RBTC, ETH_FEATURES),
```

Comment:
I was expecting this to be RSK, how come we are going with RBTC instead?

Reply:
I think RBTC vs RSK is the same question that I posed above with CELO vs CGLD. I'm curious your thoughts

Reply:
Those are the official tickers and how they will be promoted in places like coinmarketcap. Since we support assets, I think it makes sense to use the asset ticker.